pootle/apps/pootle_store/models.py

```diff
@@ -141,27 +141,13 @@ def _target(self, value):
     def _source(self):
         return self.unit._source
 
-    @property
-    def translator_comment(self):
-        return self.translator_comment_f
-
-    @translator_comment.setter
-    def translator_comment(self, value):
-        self.translator_comment_f = value
-        self._set_hash()
-
     # # # # # # # # # # # # # #  Methods # # # # # # # # # # # # # # # # # # #
 
     def __unicode__(self):
         return unicode(self.target)
 
     def _set_hash(self):
```

Comment:
What is left of this method now can be folded into the single place where it is used, isn't it?